### DISCUSSION TITLE
Markdown alert icons are not rendered in GitHub Feeds

### DISCUSSION THREAD

**Discussion by CSalih** [community/community#69601] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Bug

### Body

[Markdown alert](https://github.com/orgs/community/discussions/16925) icons are not rendered in the feed, see screenshot.

## Screenshot
![image](https://github.com/community/community/assets/17460013/a0bb1d9b-4cdd-418d-8db2-27654b041389)
Release link: https://github.com/louislam/uptime-kuma/releases/tag/1.23.3

### Edit:
Icons are rendered everywhere else but not feed.
![image](https://github.com/community/community/assets/17460013/aaa1e63f-3b35-440a-aa3c-8ff3290bdb0e)

**Comment by moayad-star** [comment]
I think it has to do with JavaScript or a browser plugin

Try allowing JavaScript or try opening the link in another browser and give me a picture of what happens

**Reply by CSalih** [reply]
The screenshot above is from Firefox 118.0.1 (64-bit) and i do have the same issue on Google Chrome Version 117.0.5938.149 (Official Build) (64-bit) running on Ubuntu 22.04.3 LTS. JavaScript is enabled and should also be allowed.

**Reply by moayad-star** [reply]
If you have not found any solution yet, try using the latest version of the browser 

If you are desperate and do not find a solution, try deleting it and reinstalling it again

**Reply by Annush73** [reply]
> I think it has to do with JavaScript or a browser plugin
> 
> 
> 
> Try allowing JavaScript or try opening the link in another browser and give me a picture of what happens

**Reply by Annush73** [reply]
> > I think it has to do with JavaScript or a browser plugin
> > 
> > 
> > 
> > Try allowing JavaScript or try opening the link in another browser and give me a picture of what happens
> 
>

**Reply by CSalih** [reply]
I just installed Brave Version 1.58.137 Chromium: 117.0.5938.153 (Official Build) (64-bit) but same issue. Maybe something related to the OS as i installed the OS 1 week ago, all Browsers are installed with the latest version.

**Comment by AlejandroIt2019** [comment]
Could it be due to the browser support? I'm using Brave.

![image](https://github.com/community/community/assets/76499932/88a34731-8aa7-492e-b38f-bcbfe2ee1bc2)

**Reply by CSalih** [reply]
@AlejandroIt2019  this does not look like the feed view, it only happens there. I updated the discussion therefore.
